client/my-sites/email/titan-set-up-mailbox/index.tsx

```diff
@@ -56,10 +65,10 @@ const TitanSetUpMailbox = ( { selectedDomainName, source } ) => {
 
 	return (
 		<>
-			{ selectedSite && <QuerySiteDomains siteId={ selectedSite.ID } /> }
+			{ selectedSiteId && <QuerySiteDomains siteId={ selectedSiteId } /> }
 
 			<Main wideLayout={ true }>
-				<DocumentHead title={ titleCase( title ) } />
+				<DocumentHead title={ title } />
```

Comment:
There's no type definition for that module since it's an older one and seemingly hasn't been used in TS before. 
I figured we could achieve the same in CSS since it's such a trivial case.

However I have reconsidered and added a local type definition file that declares that module as `any`. I hope that's okay @Automattic/team-calypso ?

Reply:
> FWIW we can't style the title of the document with CSS - it's fixed by the browser, and specifically, the OS we're using

Oh, sorry about the confusion. My reference to styling was to the `<SectionHeader/>`

Reply:
> but can we make it a bit more specific than just any? It seems to be a straightforward case that just accepts strings and returns strings?

Sure. Thanks for the quick feedback!